src/web/static/web/css/style.css

```diff
@@ -209,6 +209,16 @@ span.no-wrap {
     z-index: -1;
 }
 
+input[type=checkbox]#image-clear_id {
+    display: inline;
+    position: relative;
+    top: 7px;
+    margin-left: 10px;
+}
+
+label[for=image-clear_id] {
+    display: inline !important;
+}
```

Comment:
This was done because the CSS code in question applies to the base form, which uses the style.css file because it is used different places across the webpage. I added it because when the form is used by an update view, and the model instance has an image attached to it, the form is displayed in a very unlogical way. The checkbox for removing the image is displayed after the file name of the image, while "remove" is on the next line.
Since I now switched from having an image field on my model, to simply using RichTextUploadingField instead of Text field, this is no longer an issue in the lore update form, but it might still be an issue in other update forms.

Reply:
Ah right, while partly skimming through the code, I thought this was specific to the lore pages 😅 In that case, the issue can be more localized to [the `semantic_file.html` template](https://github.com/MAKENTNU/web/blob/a90ac79f5756721c9a3864658a87fa62633dbc6c/src/web/templates/web/forms/widgets/semantic_file.html), and so this code should be moved to [`semantic_file.css`](https://github.com/MAKENTNU/web/blob/a90ac79f5756721c9a3864658a87fa62633dbc6c/src/web/static/web/css/forms/widgets/semantic_file.css).

Reply:
Oh wait, the lore form doesn't use the Fomantic UI widgets, as far as I can see :o Also, I don't think `semantic_file.html` currently supports clearing a submitted value, so I guess it's easiest not to worry about those widgets, and instead move the CSS code to a new CSS file that can be linked through [a `Media` class](https://docs.djangoproject.com/en/4.2/topics/forms/media/) added to all forms that use Django's vanilla file widget (which currently seems like only applies to `LoreForm`).

Ideally, `semantic_file.html` should be changed to support clearing a submitted value, but that might be out of scope of this PR.. Also I've already done it in a local branch, which I'll push sometime later ™️ 🤠

Reply:
But then everyone needing a form would have to include the media class with the correct css-file in their forms, which they will probably not do if they don't know about this separate CSS-file. Can't I just move this code to a separate file, and include this new file in the generic_form.html template instead?

Reply:
All files linked through widgets' and forms' `Media` class are automatically linked when inserting the form's `media` property into the form template (see [how the `media` property works](https://docs.djangoproject.com/en/stable/topics/forms/media/#media-on-forms)) - which should always be done when working with a form. When inheriting from `CustomFieldsetFormMixin` or otherwise using the `generic_form.html` template, [`form.media` is automatically inserted](https://github.com/MAKENTNU/web/blob/a90ac79f5756721c9a3864658a87fa62633dbc6c/src/web/templates/web/generic_form.html#L8) 🙂

However, seeing as you removed the `image` field, nevermind this 😅